content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md

```diff
@@ -3,30 +3,31 @@ title: Impact of the CircleCI Security Incident on the Datadog Agent
 kind: faq
 ---
 
-On January 4th, 2023, Datadog was notified by CircleCI that they were investigating a [security incident][1] that may have led to leaking stored secrets. Datadog identified a single secret stored in CircleCI that could theoretically be misused by a potential attacker, an old RPM GNU Privacy Guard (GPG) private signing key and its passphrase. This page documents the implications of the potential leak and the measures Datadog is taking to mitigate any risks to our customers.
+On January 4th, 2023, Datadog was notified by CircleCI that they were investigating a [security incident][1] that may have led to leaking of stored secrets. Datadog identified a single secret stored in CircleCI that could theoretically be misused by a potential attacker, an old RPM GNU Privacy Guard (GPG) private signing key and its passphrase. This page documents the implications of the potential leak and the measures Datadog is taking to mitigate any risks to our customers.
 
 <div class="alert alert-info">
-<strong>Note</strong>: As of January 12th, 2023, Datadog has no indication that the key was actually leaked or misused, but we are still taking the following actions out of an abundance of caution.
+<strong>Note</strong>: As of January 16th, 2023, Datadog has no indication that the key was actually leaked or misused, but we are still taking the following actions out of an abundance of caution.
 </div>
 
 ## The affected key
 
-The RPM GPG signing key has fingerprint `60A389A44A0C32BAE3C03F0B069B56F54172A230`, and is accessible in [our signing keys location][2]. This key was historically used to sign Agent 5 releases and Agent 6 releases up to (and including) 6.13.0.
+The impacted RPM GPG signing key has the fingerprint `60A389A44A0C32BAE3C03F0B069B56F54172A230`, and is accessible in [our signing keys location][2]. This key was historically used to sign Agent 5 releases, Agent 6 releases up to (and including) v6.13.0, DogStatsD 6 releases and DogStatsD 7 releases up to (and including) v7.23.1.
 
 <div class="alert alert-info">
-<strong>Note</strong>: Official Datadog repositories were <strong>not</strong> compromised. The signing key, if actually leaked, could be used to construct an RPM package that looks like it's from Datadog but it would not be enough to place such a package in our official package repositories.
+<strong>Note</strong>: Official Datadog repositories were <strong>not</strong> compromised. The signing key, if actually leaked, could be used to construct an RPM package that looks like it is from Datadog but it would not be enough to place such a package in our official package repositories. Installing a malicious package on a system would require that the attacker has root access to the system or that they can upload the package to a repository trusted by the system.
```

Comment:
I think this additional sentence adds more confusion 🤔 

An attacker with root access to a system could install absolutely everything on the system, regardless of whether a key was leaked or not, so I don't think it's relevant to even mention it. That said, I think it is relevant to say that an attacker would need to be able to upload the package to a repository _used_ by the system.

So I'd suggest the following (but I'm not even sure it's better, it might still add confusion, happy to get some more feedback on it):

```suggestion
<strong>Note</strong>: Official Datadog repositories were <strong>not</strong> compromised. The signing key, if actually leaked, could be used to construct an RPM package that looks like it is from Datadog but it would not be enough to place such a package in our official package repositories. A hypothetical attacker with the affected key would need to be able upload the constructed RPM package to a repository used by the system.
```